widgets/filters/src/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface Filter {
+    id: string;
+    // eslint-disable-next-line camelcase
+    created_by: string;
+    // eslint-disable-next-line camelcase
+    created_at: string;
+}
+
+export interface WidgetConfiguration {
+    pageSize: number;
+}
+
+export type Widget = Stage.Types.Widget<WidgetConfiguration>;
```

Comment:
I'm worried that such generally named types (without any `Filters` prefix) will start appearing globally when using intellisense, e.g.


```
function acceptsWidget(widget: Widget
```

and if you ran some kind of "auto import", it could import the file from here.

I would encourage naming them with a prefix

```suggestion
export interface FiltersWidgetConfiguration {
    pageSize: number;
}

export type FiltersWidget = Stage.Types.Widget<WidgetConfiguration>;
```

Reply:
I agree, changed.